main.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "planet"
+require_relative "solar_system"
+
+
+def main
+    earth = Planet.new("Earth", "blue-green", 5.972e24,1.496e8, "Earth is the only planet in our solar system not to be named after a Roman or Green deity.")
+    mars = Planet.new("Mars", "red", 6.39e23, 227940000, "Olympus Mons, the tallest mountain in the solar system, calls Mars home.") 
+    venus = Planet.new("Venus", "yellow", 4.867e24, 1.089e8, "Venus spins clockwise on its axis.")
+    mercury = Planet.new("Mercury", "yellow-blue", 3.285e23, 5.791e7, "Your weight on Mercury would be 38% of your weight on Earth.")
+    pluto = Planet.new("Pluto", "grey-orange", 1.30900e22, 5.906e9, "Named after the Roman god of the underworld.")
+
+
+    solar_system = SolarSystem.new("Sol")
+    solar_system.add_planet(earth)
+    solar_system.add_planet(mars)
+    solar_system.add_planet(venus)
+    solar_system.add_planet(mercury)
+    solar_system.add_planet(pluto)
+    list = solar_system.list_planets
+    puts list
+
+    until directions != "exit"
```

Comment:
You should also initialize `directions`, i.e. give it a start value.

```suggestion

    directions = ""
    until directions != "exit"
```